webcompat/static/js/lib/wizard/steps/screenshot.js

```diff
@@ -16,7 +21,7 @@ import {
 } from "../validation.js";
 import { showContainer } from "../ui-utils.js";
 import { downsampleImage, convertToDataURI } from "../utils.js";
-import uploadHelper from "./upload-helper/index.js";
```

Comment:
I've renamed these methods to distinguish between "saving" a image as a background and the actual upload that happens on form submit